unittests/abi_tests.cpp

```diff
@@ -40,7 +40,17 @@ FC_REFLECT(act_sig, (sig) )
 
 BOOST_AUTO_TEST_SUITE(abi_tests)
 
+#ifdef NDEBUG
 fc::microseconds max_serialization_time = fc::seconds(1); // some test machines are very slow
+#else
+fc::microseconds max_serialization_time = fc::microseconds::maximum(); // don't check in debug builds
+#endif
```

Comment:
Was this because it was failing locally on a debug build? I think just increasing the existing to 2 or 3 seconds would be preferred to having diff behavior for debug.

Reply:
No, it was because the timeouts were hit when I was stepping through the code. The debug build purpose is for debugging, so having timeouts is unfriendly for that). Is there a reason why timeouts in abi tests would be useful in debug builds?

Reply:
Ah, that makes sense. There is a potential for a hang with a debug build, but one could always attach a debugger in that case.

Reply:
It might be helpful to add a comment explaining in the test code why #if NDEBUG is required.